src/components/Menu/MenuItem.tsx

```diff
@@ -71,7 +99,12 @@ export const MenuItem = React.forwardRef<HTMLButtonElement, MenuItemProps>(
             height={18}
           />
         )}
-        {!!text && text}
+        <Box direction="column" align="flex-start" gap={0.25}>
+          <Box className={classes.text}>{!!text && text}</Box>
+          <Box className={classes.secondaryText}>
+            {!!secondaryText && secondaryText}
+          </Box>
```

Comment:
Should we render this `div` even if there is no `secondaryText` passed in or should we conditionally render this whole section.

```suggestion
         {!!secondaryText && (
            <Box className={classes.secondaryText}>
               {secondaryText}
            </Box>
          )}
```

Reply:
Agreed on this nit